### ISSUE TITLE
[QUESTION] Training my own metric using Comet regression model

### ISSUE THREAD

**Issue by xu1998hz** [Unbabel/COMET#46]
Hi Ricardo, I read all your code implementations. From my understanding, when I use comet-train to train my own evaluation metric, I will not load in any pretrained Comet weights except the initialization of XLM-Roberta weights. I just want to double check if this is correct. Currently, I set "resume_from_checkpoint" in the trainer.yaml in the config to be "null".

Thank you in advance! This is a great work.

**Comment by ricardorei** [comment]
Yes, typically when you use `comet-train` you are just loading XLM-R weights (or any other encoder) and finetuning on top.

The only exception is if you use the model flag `load_weights_from_checkpoint`. This flag can be used to load an existing COMET checkpoint.

**Comment by ricardorei** [comment]
For our models from WMT21 shared task we actually did that.. we train a checkpoint on DA's and then we fine-tune them in MQM z-scores using that same flag: [Are References Really Needed?
Unbabel-IST 2021 Submission for the Metrics Shared Task](https://statmt.org/wmt21/pdf/2021.wmt-1.111.pdf)

**Comment by ricardorei** [comment]
Btw the advantage of `load_weights_from_checkpoint` over `resume_from_checkpoint` is that the architecture doesn't need to be exactly the same! we only load the matching (key, values) between the new model and the checkpoint. While in `resume_from_checkpoint` the model architecture needs to be exactly the same. Does this make sense?

Let's imagine you want to reuse the weights learned with a Ranking Model... with `load_weights_from_checkpoint` you will be able to do that and then fine-tune a Regression Model.

**Comment by xu1998hz** [comment]
Thank you so much for your detailed explanations!
